parlai/core/torch_ranker_agent.py

```diff
@@ -883,7 +945,9 @@ def _make_candidate_vecs(self, cands):
         cand_vecs = []
         for batch in tqdm(cand_batches):
             cand_vecs.extend(self.vectorize_fixed_candidates(batch))
-        return padded_3d([cand_vecs], dtype=cand_vecs[0].dtype).squeeze(0)
+        return padded_3d(
```

Comment:
😮 good catch

Reply:
credit for this goes to @dexterju who had a bug with this (and has probably fixed in his local checkout)